war/src/main/webapp/css/style.css

```diff
@@ -1913,3 +1913,18 @@ body.no-sticker #bottom-sticker {
     width: 48px;
     height: 48px;
 }
+
+/* rss-bar */
+
+.icon-rss {
+    border: 0;
+}
+
+.rss-bar-item {
+    padding-left: 1em;
+}
+
+#rss-bar {
+    margin:1em;
+    text-align:right;
+}
```

Comment:
```
/* rss-bar */

#rss-bar {
    margin: 1em;
    text-align: right;
}

#rss-bar .icon-rss {
    border: 0;
}

#rss-bar .rss-bar-item {
    padding-left: 1em;
}
```

More robust.

Reply:
Finally this change request was not addressed.

Reply:
@recena it was addressed. Please look at the final code.